lib/ansible/executor/task_executor.py

```diff
@@ -920,6 +920,8 @@ def _get_connection(self, variables, templar):
         )
 
         if not connection:
+            print(plugin_load_context.redirect_list)
```

Comment:
I'm guessing these prints were not supposed to be left in?

Reply:
What ever do you mean? ;)